tests/scripts/helm.sh

```diff
@@ -26,19 +24,31 @@ detectArch() {
     esac
 }
 
-install() {
-    if ! helm_loc="$(type -p "helm")" || [[ -z ${helm_loc} ]]; then
-        # Download and unpack helm
-        local dist
-        dist="$(uname -s)"
+# Echo the name or path of the helm command. Install helm to a temp location
+# if it's not present.
+install_helm() {
+    local helm="helm"
```

Comment:
Why are you using a local variable instead of overriding the variable and later on calling the function?

Reply:
I understood an earlier comment to suggest not to define a global var in this func. I agree (to improve structure and readability) and so I now use a local var to hold the actual helm cmd, then return its value  (via stdout) and the main script assigns it to the HELM var.

Reply:
I see, my first comment was meant that the variable from line 6 is used and the function just overrides it when helm is not installed.

Reply:
HELM now is only assigned to in one place, so it's easy to know its value. Are you ok with this?